docs/outlook/delegate-access.md

```diff
@@ -79,7 +79,11 @@ The following example shows the `SupportsSharedFolders` element set to `true` in
 You can get an item's shared properties in Compose or Read mode by calling the [item.getSharedPropertiesAsync](../reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md#methods) method. This returns a [SharedProperties](/javascript/api/outlook/office.sharedproperties) object that currently provides the delegate's permissions, the owner's email address, the REST API's base URL, and the target mailbox.
 
 > [!IMPORTANT]
-> In a delegate scenario, your add-in can use REST but not EWS, and the add-in's permission must be set to `ReadWriteMailbox` to enable REST access to the owner's mailbox.
+> - In a delegate scenario, your add-in can use REST but not EWS, and the add-in's permission must be set to `ReadWriteMailbox` to enable REST access to the owner's mailbox.
+> - In Message Compose mode, [getSharedPropertiesAsync](/javascript/api/outlook/office.messagecompose#getsharedpropertiesasync-options--callback-) is not supported in Outlook on the web or Windows unless the following conditions are met.
+>    1. The owner shares at least one mailbox folder with the delegate.
+>    2. The delegate saves a draft message then moves it from their own **Drafts** folder to the shared folder.
```

Comment:
Ideally they don't need to move it, if you hit reply in your inbox and hit ctrl+s, there will be a draft shown in the inbox itself too but I believe this is still ok